### PR TITLE
Create a static page that displays text content and main portrait 

### PR DESCRIPTION
**Pull request by Aradhya-B**
image

This PR creates the HTML skeleton for my portfolio that displays a brief timeline, a portrait configured with the "Breathing Half Tone" library (https://breathing-halftone.desandro.com/), icons that link to my social medias and contact, and uses styling to format and make the single page responsive. 

### Responsiveness 

image
image

### CODE REVIEW

File: portfolio/src/main/webapp/assets/icons/secret.svg [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+<path style="fill:#E5D8C6;" d="M380.211,86.329V3.812l99.267,99.267H396.96C387.71,103.079,380.211,95.58,380.211,86.329z"/>
+<g>
+	<path style="fill:#FFFFFF;" d="M428.69,127.932H83.31c-4.278,0-7.747,3.468-7.747,7.747s3.468,7.747,7.747,7.747h345.38
+		c4.278,0,7.747-3.468,7.747-7.747C436.437,131.4,432.968,127.932,428.69,127.932z"/>
+	<path style="fill:#FFFFFF;" d="M83.31,90.403h246.025c4.278,0,7.747-3.468,7.747-7.747s-3.468-7.747-7.747-7.747H83.31
+		c-4.278,0-7.747,3.468-7.747,7.747C75.563,86.935,79.032,90.403,83.31,90.403z"/>
+</g>
+<g>
+</g>
+<g>
```
Comment (nickKelman) [inline portfolio/src/main/webapp/assets/icons/secret.svg:62]: Are all these opening and closing tags necessary?

File: portfolio/src/main/webapp/assets/icons/secret.svg [hunk 2]
```diff
@@ -0,0 +1,90 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
+<!-- Generator: Adobe Illustrator 19.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="Capa_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
```
Comment (nickKelman) [inline portfolio/src/main/webapp/assets/icons/secret.svg:3]: It seems like the dom elements aren't indented properly. I assume this is copy pasta from somewhere?

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -1,18 +1,142 @@
...
+<html lang="en">
   <head>
-    <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
+    <meta charset="UTF-8" />
+    <link rel="shortcut icon" href="assets/cartoon-edited.png" />
+    <link rel="stylesheet" type="text/css" href="styles/styles.css" />
+    <link
```
Comment (nickKelman) [inline portfolio/src/main/webapp/index.html:7]: nit: typically we put the first element attribute so that it is on the same line as the starting tag.

Comment (nickKelman) [reply portfolio/src/main/webapp/index.html:7]: Applies elsewhere

File: portfolio/src/main/webapp/index.html [hunk 4]
```diff
@@ -1,18 +1,142 @@
...
+            McGill
+          </li>
+          <li>
+            '18: Didn't win large scholarships at U Waterloo, U of T, UBC, U of
+            A, or U of C
+          </li>
+          <li>
+            '18: Joined
+            <a class="list-link" href="https://gdmissionsystems.ca/" target="#"
+              >General Dynamics</a
```
Comment (nickKelman) [inline portfolio/src/main/webapp/index.html:67]: A lot of the closing anchor tags in this file seem off

File: portfolio/src/main/webapp/index.html [hunk 5]
```diff
@@ -1,18 +1,142 @@
...
+            self-improvement. 4. dreams big and gets s#$% done
+          </em>
+        </h5>
+        <p class="main">
+          <em>"Vulnerability doesn't come after trust - it precedes it."</em> ~
+          <a
+            class="list-link"
+            href="https://fourminutebooks.com/the-culture-code-summary/"
+            target="#"
+            ><u>The Culture Code:</u></a
```
Comment (cftorres) [inline portfolio/src/main/webapp/index.html:38]: Nit: I would out this end bracket on the same line as above and start and finish the </a> on a new line (here and applies to similar blocks below)

File: portfolio/src/main/webapp/scripts/script.js [hunk 6]
```diff
@@ -13,16 +13,26 @@
...
 /**
- * Adds a random greeting to the page.
+ * Plays audio clip
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+const playAudio = (audioClip) => {
+	const audio = new Audio(audioClip);
+	audio.volume = 0.5;
```
Comment (nickKelman) [inline portfolio/src/main/webapp/scripts/script.js:20]: Don't be shy

File: portfolio/src/main/webapp/styles/styles.css [hunk 7]
```diff
@@ -0,0 +1,118 @@
...
+}
+
+main {
+    display: flex;
+    flex-basis: 50%;
+    flex-wrap: wrap;
+    justify-content: center;
+    width: 100%;
+    height: auto;
+    margin: 1em auto;
```
Comment (cftorres) [inline portfolio/src/main/webapp/styles/styles.css:16]: Nit: make consistent indentation across page

**Review by cftorres — APPROVED** [review]

File: portfolio/src/main/webapp/styles/styles.css [hunk 8]
```diff
@@ -0,0 +1,118 @@
...
+}
+
+.title {
+    color: #0c8716;
+    font-size: 4em;
+    margin: 0;
+}
+
+#pronounce {
+  width: 1.5em;
```
Comment (nickKelman) [inline portfolio/src/main/webapp/styles/styles.css:35]: nit: indentation in this file should be consistent

File: portfolio/src/main/webapp/styles/styles.css [hunk 9]
```diff
@@ -0,0 +1,118 @@
...
+
+.list-link:hover {
+    color: #0c8716;
+}
+
+footer {
+  text-align: center;
+}
+
+/* Responsive layout - when the screen is less than 700px wide, make the two columns stack on top of each other instead of next to each other */
```
Comment (nickKelman) [inline portfolio/src/main/webapp/styles/styles.css:101]: Very nice! Can you add a screenshot of this behavior to the description?

**Review by nickKelman — COMMENTED** [review]
Very nice work. Thanks for taking the time to make this first review smaller.

File: portfolio/src/main/webapp/index.html (hunk 5 above)
Comment (Aradhya-B) [reply portfolio/src/main/webapp/index.html:38]: Got it, I think a lot of these earlier commits were pretty spaghetti so I apologize for that.

File: portfolio/src/main/webapp/assets/icons/secret.svg (hunk 2 above)
Comment (Aradhya-B) [reply portfolio/src/main/webapp/assets/icons/secret.svg:3]: Yeah this is just an svg I downloaded and added to my assets folder. I did not write any of this code haha.

File: portfolio/src/main/webapp/styles/styles.css (hunk 7 above)
Comment (Aradhya-B) [reply portfolio/src/main/webapp/styles/styles.css:16]: Will do.

File: portfolio/src/main/webapp/index.html (hunk 3 above)
Comment (Aradhya-B) [reply portfolio/src/main/webapp/index.html:7]: Got it.

File: portfolio/src/main/webapp/index.html (hunk 4 above)
Comment (Aradhya-B) [reply portfolio/src/main/webapp/index.html:67]: Yeah I think my linter in Vim was off or something, I think all of that is fixed by the last commit, unfortunately a lot of these earlier ones have a lot of spaghetti code.

File: portfolio/src/main/webapp/styles/styles.css (hunk 8 above)
Comment (Aradhya-B) [reply portfolio/src/main/webapp/styles/styles.css:35]: will do

File: portfolio/src/main/webapp/styles/styles.css (hunk 9 above)
Comment (Aradhya-B) [reply portfolio/src/main/webapp/styles/styles.css:101]: Done.

**Pull request merged**